packages/salesforce-adapter/src/filters/changed_at_singleton.ts

```diff
@@ -59,6 +64,12 @@ const getChangedAtSingletonInstance = async (
   return changedAtSingleton ?? createEmptyChangedAtSingletonInstance()
 }
 
+const dateStringOfMostRecentlyChangedInstance = (instances: InstanceElement[]): string => (
+  _(instances)
+    .map(instance => instance.annotations[CORE_ANNOTATIONS.CHANGED_AT])
+    .maxBy((changedAt: string) => new Date(changedAt).getTime())
```

Comment:
```suggestion
  _(instances)
    .map(instance => instance.annotations[CORE_ANNOTATIONS.CHANGED_AT])
    .filter(changedAt => changedAt !== undefined && changedAt !== '')
    .maxBy((changedAt: string) => new Date(changedAt).getTime())
```
Let's filter empty or undefined values, to avoid problem with creating a `javascript Date` object from them.

Reply:
Ack